scalding-commons/src/main/scala/com/twitter/scalding/commons/source/LzoGenericScheme.scala

```diff
@@ -102,15 +118,7 @@ class LzoGenericScheme[M](@transient conv: BinaryConverter[M], clazz: Class[M])
     tap: Tap[JobConf, RecordReader[_, _], OutputCollector[_, _]],
     conf: JobConf): Unit = {
 
-    val extern = Externalizer(conv)
-    try {
-      ExternalizerSerializer.inj.invert(ExternalizerSerializer.inj(extern)).get
-    } catch {
-      case e: Exception => throw new RuntimeException("Unable to roundtrip the BinaryConverter in the Externalizer.", e)
-    }
-
-    conf.set(ConfigBinaryConverterProvider.ProviderConfKey, ExternalizerSerializer.inj(extern))
-
+    LzoGenericScheme.setConverter(conv, conf)
```

Comment:
So we don't want to set the converter here when it's already been set earlier. I'm guessing this is because we want it to be configurable via the new method, before this scheme code is executed?